src/instructlab/config/init.py

```diff
@@ -378,8 +391,9 @@ def get_params(
     if taxonomy_contents:
         clone_taxonomy_repo = False
     elif interactive and param_source != click.core.ParameterSource.ENVIRONMENT:
+        click.echo(f"`{taxonomy_path}` seems to not exist or is empty.")
         clone_taxonomy_repo = click.confirm(
-            f"`{taxonomy_path}` seems to not exist or is empty. Should I clone {repository} for you?",
```

Comment:
Debating whether this is an improvement.  I would worry the new line will cause some people to miss the context above it.

Reply:
Personally, compared the old display, it is hard to read, even though show it one by one. Actually, I just try to separate the different `prompt` and `confirm`. if the same prompt, should be together, no new line for it.
```
# ilab config init
Found /root/.config/instructlab/config.yaml, do you still want to continue? [y/N]: y
Existing training profiles were found in /root/.local/share/instructlab/internal/train_configuration/profiles
Do you want to restore these profiles to the default values? [y/N]: y
Welcome to InstructLab CLI. This guide will help you to setup your environment.
Please provide the following values to initiate the environment [press Enter for defaults]:
Path to taxonomy repo [/root/.local/share/instructlab/taxonomy]:
Path to your model [/root/.cache/instructlab/models/merlinite-7b-lab-Q4_K_M.gguf]:

====================================================
-----------------------------------------------------
         Welcome to InstructLab CLI.
  This guide will help you to setup your environment.
-----------------------------------------------------

Please provide the following values to initiate the environment [press Enter for defaults]: <<<
Path to taxonomy repo [/root/.local/share/instructlab/taxonomy]:  <<<
Path to your model [/root/.cache/instructlab/models/merlinite-7b-lab-Q4_K_M.gguf]:  <<<

Generating config file and profiles:
    /root/.config/instructlab/config.yaml
    /root/.local/share/instructlab/internal/train_configuration/profiles
```

Reply:
After pulling the code and running init, I no longer have a concern.  The extra whitespace between sections makes it clear you need to read the lines above up to the whitespace.